nss-cohort-36/bangazon-client-virtual-magicians-1
# Jl 2nd react ticket

# Description
[Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.]
Closes #[Enter issue #]
## Type of change
new feature - customer now can have an address - and change their address as well (address is only city name)

Please delete options that are not relevant.

# Testing Instructions
Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout jl-2nd-reactTicket`
4. 
you will need to have the api branch down and tables updated prior to this being functional
once this is done, click on MyProfile in navbar - click the update location button - add a location to your user - check tableplus to see the location has changed

# Checklist:
